node/pkg/websocketfetcher/providers/bybit/bybit.go

```diff
@@ -28,8 +29,8 @@ func New(ctx context.Context, opts ...common.FetcherOption) (common.FetcherInter
 	}
 
 	subscriptions := []any{}
-	for i := 0; i < len(pairList); i += 3 {
-		end := common.Min(i+3, len(pairList))
+	for i := 0; i < len(pairList); i += 10 {
+		end := common.Min(i+10, len(pairList))
```

Comment:
hmm I curious what are these numbers (i + 10 and the 10 in the for loop) based on... I'm wondering if its worth declaring them as global constants with a brief explanation comment for clarity?

Reply:
it is based on the official documentation of bybit, I'll add comment about this
https://bybit-exchange.github.io/docs/v5/ws/connect#public-channel---args-limits